packages/gestalt/src/ModalAlert/ModalAlertAction.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
```

Comment:
Mostly just moving these components around — no material changes to either aside from adding `dataTestId` to this one